baker/postUpdatedHook.ts

```diff
@@ -19,21 +19,58 @@ const syncPostToGrapher = async (
 ): Promise<string | undefined> => {
     const rows = await wpdb.singleton.query(
         `--sql
-        -- we select all the fields from wp_posts and then we also
+        -- This query extracts all the fields from the wp_posts table and then
+        -- adds a json array of authors, and a created_at field which is
+        -- constructed by finding the revisions related to this post/page and
+        -- taking the earliest one post_date.
+
+        -- We use a CTE to find the all revisions for each post/page
+        with revisions as (
+            select
+                post_date,
+                post_parent,
+                row_number() over
+                        ( order by post_date) row_num
+            from
+                wp_posts
+            where
+                post_type = 'revision' and post_parent = ?
+        ),
+        -- and then we use another CTE to find the first revision
+        first_revision as (
+            select
+                post_date as created_at,
+                post_parent as post_id
+            from
+                revisions
+            where
+                row_num = 1),
```

Comment:
would that not be easier with the `LIMIT 1`, as ike had initially suggested?

Reply:
Ah nice, in this one that is definitely the better choice. I wrote the other one first and then this was the simple adaptation but you're right, better to make it simpler. Thanks!